symengine/cwrapper.cpp

```diff
@@ -1148,6 +1148,25 @@ CWRAPPER_OUTPUT_TYPE dense_matrix_eye(CDenseMatrix *s, unsigned long int N,
     CWRAPPER_END
 }
 
+CWRAPPER_OUTPUT_TYPE dense_matrix_diff(CDenseMatrix *result,
+                                       const CDenseMatrix *A, basic const x)
+{
+    if (not is_a_Symbol(x))
+        return SYMENGINE_RUNTIME_ERROR;
```

Comment:
@ShikharJ `is_a_Symbol` gets tested several times, so how is one supposed to test a define like `SYMENGINE_RUNTIME_ERROR` ?